nucleus/async_job.py

```diff
@@ -131,6 +131,27 @@ def from_json(cls, payload: dict, client):
         )
 
 
+class EmbeddingsExportJob(AsyncJob):
+    def result_urls(self) -> List[str]:
+        """Gets a list of signed Scale URLs for each embedding batch.
+
+        Returns:
+            A list of signed Scale URLs which contain batches of embeddings.
+
+            The files contain a JSON array of embedding records with the following schema:
+                [{
+                    "reference_id": str,
+                    "embedding_vector": List[float]
+                }]
+        """
+        status = self.status()
+
+        if status["status"] != "Completed":
+            raise JobError(status, self)
+
```

Comment:
why raise a `JobError` if the job is not completed? perhaps its still running?

Reply:
My thought process was that the usage pattern would be the following:

```python
exportJob = dataset.export_embeddings()
exportJob.sleep_until_complete(False)
result = exportJob.result_urls()
```

We could just wait for the result urls inside `result_urls()` also, but then I'd highlight it somehow that obtaining the results could run for a long time.

Reply:
Alright, that makes sense, didn't noticed the AsyncJob inheritence.
This is a neat idea, to have customized job result classes

Reply:
Maybe we should add a `wait_for_completion` parameter. It might even be the default to wait for the job to complete.

Reply:
Good idea, let's do that